tasking/reaper.go

```diff
@@ -167,7 +156,11 @@ func (r *TaskReaper) deletePod(m *model.Task) (err error) {
 //
 // delete task.
 func (r *TaskReaper) delete(m *model.Task) {
-	err := r.DB.Delete(m).Error
+	err := r.deletePod(m)
+	if err != nil {
+		Log.Trace(err)
+	}
+	err = r.DB.Select(clause.Associations).Delete(m).Error
```

Comment:
Add the Select(clause.Associations) so the ORM will delete the associated TaskReport.